canonicalwebteam/blog/helpers.py

```diff
@@ -128,22 +126,33 @@ def transform_article(article, group=None):
             article["_end_day"], end_month_name, article["_end_year"]
         )
 
+    if transform_links:
+        article = recursive_search_and_replace(article)
+
     return article
 
 
-def change_url(feed, host):
-    """Change insights urls to <host>/blog
+def recursive_search_and_replace(article):
```

Comment:
Do we need recursion and type checking here? I would expect we always call `transform_article` on a single article of the same type, so we should only need to replace the links on `article["content"]`(not sure that is the right key)

Reply:
I removed the recursive replace. Instead there are two places I am replacing the url manually:
- `article['content']['rendered']` for the article page content
- `article['image']['source_url']` for the thumbnails

It was an option I considered initially. I thought recursivity will cover some places I am unaware of. But it should be fine with these two.